FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -0,0 +1,964 @@
+---
+openapi: 3.0.1
+info:
+  title: FRAIHMWORK Health And Integrity API
+  description: 'FRAIHMWORK allows for multiple implementations of basic health and integrity monitoring services for other systems in a given network. This set of APIs are for systems that elect to do HTTP POST calls to push their state and fault information to the FRAIHMWORK platform, rather than hosting their own server endpoint or TCP/UDP streams. This initial version does not imply any protections against malicious behavior from trusted endpoints. This API will versioned according to SemVer 2.0 rules (as described here: https://semver.org/)'
```

Comment:
@GD-ResilienX  Looks like a new yaml file is created instead of updated the existing one. In that case, we should get rid of the existing FraihmworkHealthAndIntegrityMonitoringApi and RegistrationApi files as they are already present in the Deprecated folder.